### PR TITLE
Replaces the King Goats Guards with a Goat Pope

### PR DESCRIPTION
**Pull request by fluffe9911**
🆑 Identification for pope goat sprites Fluffe9911 for coding
rscadd: A big goat pope hat
tweak: The master guards have been let off duty and replaced with the pope goat
/🆑

### CODE REVIEW

File: yogstation/code/modules/mob/living/simple_animal/hostile/retaliate/king_of_goats.dm [hunk 1]
```diff
@@ -154,6 +154,19 @@ Difficulty: Insanely Hard
 	melee_damage_upper = 20
 	move_to_delay = 3
 
+/mob/living/simple_animal/hostile/retaliate/goat/guard/pope
+	name = "Goat Pope"
+	desc = "For what is a god without a pope to spread it's holy words"
```
Comment (HomocidlePancakes) [inline yogstation/code/modules/mob/living/simple_animal/hostile/retaliate/king_of_goats.dm]: >God
>It

bruh.

**Comment by nesisknesis** [comment]
image

**Comment by fluffe9911** [comment]
give me a second jesus icon conflicts lol

File: yogstation/code/modules/mob/living/simple_animal/hostile/retaliate/king_of_goats.dm (hunk 1 above)
Comment (fluffe9911) [reply yogstation/code/modules/mob/living/simple_animal/hostile/retaliate/king_of_goats.dm]: Archbishop of Canterbury said god is gender neutral so shush

**Comment by HomocidlePancakes** [comment]
God bless

**Comment by Amelia0010** [comment]
I'm apparently the designated goat-merger, so sure

**Pull request merged**